soda_machine.rb

```diff
@@ -7,12 +7,25 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    return @sodas.length
   end
 
   def find_soda(soda_brand)
+    soda = soda_brand.downcase.split(' ').join('_')
```

Comment:
What is the purpose of this line? At least for this challenge, you can do direct string comparison without sanitizing input, i.e. `"Mountain Dew" == "Mountain Dew"`. Perhaps there was confusion with the name of the `Soda` instances in `soda_machine_spec.rb` where you thought you had to match the local RSpec variable name (`mountain_dew`) to the `brand` string of the object (`Mountain Dew`).